app/api/v1/user.py

```diff
@@ -0,0 +1,31 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.exc import DBAPIError
+
+from app.core.database import get_db
+from app.models.user import UserRead, UserCreate
+import app.crud.user as crud_user
+
+router = APIRouter(prefix="/user", tags=["user"])
+
+
+@router.get("/get-user", response_model=UserRead | None)
+async def get_user(id: str, db: AsyncSession = Depends(get_db)):
+    try:
+        user = await crud_user.get(db, id)
+    except DBAPIError:
+        return None
+
+    return user
+
+
+@router.get("/get-users", response_model=list[UserRead])
```

Comment:
We definitely need to talk API at some point - no need to fix this now, but I'd prefer REST semantics

Reply:
updated endpoint names